src/utils.ts

```diff
@@ -1344,3 +1344,56 @@ export async function fileIsAccessible(fileName: string, mode?: number) {
     return false;
   }
 }
+
+/** @internal */
+export class MovingWindow {
```

Comment:
Correct me if I'm wrong, but the logic within this seems specific to our RTT rules, like the way less than 2 samples are handled. `class RTTSampler`?

Reply:
If this is specifically for RTT calculations and we're only using it in the monitor, can we move it out of the utils file?

Reply:
@W-A-James

Reply:
Moved to `src/sdam/monitor.ts`